### DISCUSSION TITLE
center point appears on the left side

### DISCUSSION THREAD

**Discussion by alsahime** [mriscoc/Ender3V2S1#820] [Q&A]
Hello, I am using an Ender 3 V2 with a BTT SKR Mini E3 V3 motherboard. I have installed the mriscoc firmware, but when I start the printing process, the printer only draws a straight line in the empty space and the center point appears on the left side, with the X-axis shifted about 16mm to the left. How can I edit or adjust this?

**Comment by dfinberg** [comment]
Sounds like your X minimum is not at 0.  You need to set the physical dimensions so that X=0 is the edge of the bed, so maybe your bed X minimum should be -16?  There's a section of the wiki on physical dimensions setup.

**Comment by mriscoc** [comment]
https://github.com/mriscoc/Ender3V2S1/wiki/Calibration-Guides
